src/jit/compiler.h

```diff
@@ -193,9 +193,9 @@ inline HfaElemKind HfaElemKindFromType(var_types type)
 typedef const char* VarName; // Actual ASCII string
 struct VarScopeDsc
 {
+    unsigned  vsdVarNum;  // (remapped) LclVarDsc number
```

Comment:
What does moving this do?

Reply:
thanks for catching it, it was strange formatting. 
That was a debug only warning because of 8 bytes `VarName vsdName;`.
I will move it 3 lines down (after `vsdVarNum`) so it won't look that awkward.

It doesn't affect release, but cleans the warning in my report, so I don't need to look at it each time and it is a good thing for the checked build.